include/deal.II/grid/grid_in.h

```diff
@@ -663,8 +663,25 @@ class GridIn
 
 
   DeclException0(ExcGmshNoCellInformation);
+  
+  /**
+   * 
+   */
+  std::map< std::string, std::map<int,double> > get_field_data()
+  {
+      
+      return this->field_data;
+  }
 
 protected:
+    
+  /**
+  * Data member that stores cell data. The format is as follows:
+  * std::string stores identified
```

Comment:
I don't understand this half sentence. Can you expand on it?

doxygen will not render the line breaks like you have it here. If you want this and the following line to be a bullet point list, start the line with " - ".